### ISSUE TITLE
Staging Review finding: Unlabeled radio button group

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99047]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 10-10 Health Apps
**Product:** 10-10CG
**Feature:** Improve Facility Selection

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/locator](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/veteran-information/va-medical-center/locator) after completing a search, you get a `va-radio` group containing radio button tiles for your search results. While each individual radio button is labeled, the `va-radio` group itself doesn't have a `label` prop.

The `label` prop on `va-radio` maps to a `legend` in the `fieldset` of the rendered HTML. For radio button groups, the `legend` provides instructions for what a user is supposed to do and if it's required --- in this case, select a facility. Without that instruction, most users are probably going to figure out what they're supposed to do, but it's a little extra cognitive load that we can reduce just with an extra line of text.

<details><summary>Link, screenshot or steps to recreate</summary>

![Radio tile group of VA medical facilities with no instruction text."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51980-Brian-2.png?raw=true)

</details>

## Recommended action

Recommend including something in the `label` prop for the `va-radio`, eg. "Select a VA medical center or clinic (* Required)."

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jessicastump** [comment]
This is addressed in the updated designs / content we work on with our CAIA partner post-research study. We did add a header label to the radio button list and updated the header.
- [Design ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97697)

**Comment by hdjustice** [comment]
Completed with #99135
